app/services/prison_visits/api.rb

```diff
@@ -34,7 +34,9 @@ def validate_prisoner(number:, date_of_birth:)
       result.fetch('validation')
     end
 
-    def get_slots(prison_id:, prisoner_number:, prisoner_dob:, use_nomis_slots: false)
+    def get_slots(
```

Comment:
Disabling only for method declarations would be nice, I doubt rubocop would allow something like that though.

I guess I prefer to see them formatted as a hash, so each key in a new line if they aren't going to fit into a single line, but is just personal preference... I'm ok leaving like this, hopefully there wont be anymore methods that we need to break.